commands/done.js

```diff
@@ -0,0 +1,20 @@
+const fs = require('fs');
+const tasks = JSON.parse(fs.readFileSync('./tasks.json', 'utf8'))
+
+const done = (num) => {
+  try {
+    let index = tasks.findIndex((task) => {
+      return task.id === +num
+    })
+    if (index < 0) {
+      throw new Error()
+    }
+    tasks.splice(index, 1)
+    fs.writeFileSync('./tasks.json', JSON.stringify(tasks));
+    }
+  catch(error) {
+    console.log('ID ' + num + ' does not exist')
+  }
```

Comment:
It's a little odd to explicitly throw an error and then not propagate it within a `try... catch` block. What is the desired behavior? If it is that the user is informed when they enter an invalid ID, then you either of the following alternatives would be acceptable:

```js
let index = tasks.findIndex((task) => {
  return task.id === +num
})
if (index < 0) {
  throw new Error('ID ' + num + ' does not exist')
}
```

```js
let index = tasks.findIndex((task) => {
  return task.id === +num
})
if (index < 0) {
  console.log('ID ' + num + ' does not exist')
}
```